AnagramCheck/Java/AnagramCheck.java

```diff
@@ -0,0 +1,41 @@
+public class Anagrams {
+
+  public static void main(String[] args){
```

Comment:
As these are test cases, it would perhaps be a good idea to add what strings are being verified. Currently it simply outputs their truth values.

Example:
```
true
true
false
```

Enhanced output could look like:
```
anagramCheck("Tom Marvolo Riddle", "I am Lord Voldemort") = true
anagramCheck("Hello", "olleH") = true
anagramCheck("Hi", "hello") = false
```